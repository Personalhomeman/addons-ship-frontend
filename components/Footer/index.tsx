import SVG from 'react-svg';
import css from './style.scss';

export default () => (
  <div className={css.footer}>
    <SVG src="/static/icon-bitrise-notext.svg" className={css.icon} />
    <div className={css.text}>Ship</div>
    <div className={css.text}>An official Bitrise add-on</div>
  </div>
);
