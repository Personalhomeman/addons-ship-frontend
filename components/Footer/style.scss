@import '~@/assets/style/dimensions';
@import '~@/assets/style/colors';

.footer {
  text-align: center;

  .icon {
    display: block;
    margin-bottom: 2 * $grid;

    svg {
      fill: $grape-5;
    }
  }

  .text {
    color: $grape-5;
    // TODO: replace with element from UI library
    line-height: 4 * $grid;
    font-weight: 500;
    font-style: normal;
    font-stretch: normal;
    font-size: 10px;
    letter-spacing: 0.8px;
    text-transform: uppercase;
  }
}
