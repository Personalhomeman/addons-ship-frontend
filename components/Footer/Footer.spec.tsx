import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import Footer from '.';

describe('Footer', () => {
  it('renders correctly', () => {
    const tree = toJSON(shallow(<Footer />));
    expect(tree).toMatchSnapshot();
  });
});
