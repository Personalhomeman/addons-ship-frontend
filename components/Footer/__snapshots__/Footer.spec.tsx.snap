// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<div
  className="footer"
>
  <ReactSVG
    afterInjection={[Function]}
    beforeInjection={[Function]}
    className="icon"
    evalScripts="never"
    fallback={null}
    loading={null}
    renumerateIRIElements={true}
    src="/static/icon-bitrise-notext.svg"
    wrapper="div"
  />
  <div
    className="text"
  >
    Ship
  </div>
  <div
    className="text"
  >
    An official Bitrise add-on
  </div>
</div>
`;
